run/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from storage.models import File
+from game_runner import settings
+import uuid
+
+
+# Initial scheme
+# FilePath
+#   +file : File
+#   +path : String
+#   +run : Run
+#
+# Run
+#   +token : Token
+#   +request_time : DateTime
+#   +start_time : DateTime
+#   +end_time : DateTime
+#   +input_file_paths : FilePath[]
+#   +output_file_paths : FilePath[]
+#   +log : File
+#   +game : Game
+#
+# API Methods:
+#   +run((game : Game, file_paths : FilePath[])[]) : tokens String[]
+#   +report(from_time : DateTime) : runs : Run[]
+
+class FilePath(models.Model):
+    file = File()
+    path = models.FilePathField()
```

Comment:
A game will require some files. for example it requires client_1, client_2 and server files. However, the user should not need to worry about the path of these files when we execute the game. The mapping of each name to one or more paths is done in the game definiton(e.g. client_1 --> /root/client_code/code, etc.) 

However we should not allow these paths to be random strings. The wanted files should be defined in the game definition. What we should do in file path is to map a file to one of the file-definitions in the game. Let's assume for now that the model defining a file for a game is called FileDefinition. Then, the FilePath should map the file to an instance of FileDefinition. The API should be responsible for translating the names(e.g. client_1) to the FileDefinition instance.

Regarding the validation in the api if the game requires the three files mentioned above, sending a run with a file with a different name, e.g. stupid_data, should not be validated. However implementing this is currently not possible since Game models aren't yet here.